css-ui-4/Overview.bs

```diff
@@ -1794,7 +1793,7 @@ so that CSS can be used to restyle them.
 	and a "none" value that suppresses "native" look.
 
 	However, there is evidence that this alone is not web compatible,
-	due in part to uses such as:
+	in part due to uses such as:
```

Comment:
This was better as-is.

Reply:
Hmm, that sounded strange to me, but you're the native speaker, so I trust you.